include/circt/Dialect/Verif/VerifOps.td

```diff
@@ -346,6 +349,105 @@ def FormalOp : VerifOp<"formal", [
   }];
 }
 
+def SimulationOp : VerifOp<"simulation", [
+  DeclareOpInterfaceMethods<OpAsmOpInterface, ["getAsmBlockArgumentNames"]>,
+  IsolatedFromAbove,
+  NoTerminator,
+  RegionKindInterface,
+  SingleBlockImplicitTerminator<"verif::YieldOp">,
+  Symbol,
+]> {
+  let summary = "A simulation unit test";
+  let description = [{
+    This operation defines a simulation unit test that can be automatically run
+    by various tools. To describe a test, the body of this op should contain the
+    hardware to be tested, alongside any necessary forms of stimulus generation.
+
+    ### Inputs
+
+    The body has two block arguments as input values: a "clock" signal of type
+    `!seq.clock` and an "init" signal of type `i1`. The clock signal starts at 0
+    and continuously toggles between 0 and 1 throughout the simulation. The init
+    signal starts at 1, remains 1 during a single 0-to-1 transition of the
+    clock, and then drops to 0 for the remainder of the simulation.
+
+    ### Outputs
+
+    The body must have a `verif.yield` terminator op with exactly two operands:
+
+    The first operand is a "done" signal of type `i1` which indicates the end of
+    the simulation. The simulation stops when the done signal is 1 during a
+    0-to-1 transition of the clock.
+
+    The second operand is an "exit code" signal which may be of any integer
+    type. It indicates success of a test as 0, or failure as any non-zero
+    integer. Only the zero versus non-zero value distinction is relevant to
+    indicate simulation success and must be preserved. The exact non-zero value
+    is for informational purposes only and may not be reproduced accurately by
+    simulators. Simulators may in the worst case map all non-zero values to 1,
+    reducing the exit code to a single bit indicating failure or success.
+
+    ### Schedule
+
+    The clock and init values adhere to the following schedule during
+    simulation:
+
+    | Time      | Clock | Init  | Comment
+    |-----------|-------|-------| ----
+    | t0 (>=0s) | undef | undef | Clock and init may initially be undefined.
+    | t1 (>=t0) | 0     | 1     | Initialization code (e.g., `seq.initial`, Verilog `initial` procedures) may run before or after clock and init change to their initial value.
+    | t2 (>t1)  | 1     | 1     | Single rising clock edge occurs while init is high.
+    | t3 (>t2)  | 0     | 1     | Init may stay during the falling clock edge.
+    | t4 (>=t3) | 0     | 0     | Init goes to 0 before second rising clock edge.
+    | t5 (>t4)  | 1     | 0     | Clock toggles continue indefinitely.
+    | t6 (>t5)  | 0     | 0     |
+
+    Simulation termination occurs when the done signal is 1 during a 0-to-1
+    transition of the clock. The exit code is sampled at the same time as the
```

Comment:
This sentence seems to be a duplicate from above

Reply:
Removed